www/js/survey/enketo-survey.js

```diff
@@ -53,6 +53,11 @@ angular.module('emission.survey.enketo.launch', [
     });
   }
 
+  function resetView() {
+    EnketoSurvey.getState().form.resetView();
+    $('.enketo-plugin article > form').remove();
```

Comment:
why do you have to remove the form here? Why not just reset the view? I actually don't even think you need to reset the form if you are using a modal that you remove and re-add every time since that removes the entire modal including the form automatically.